benchmarks/common/benchmarks/toy_detector_benchmark.hpp

```diff
@@ -83,6 +83,11 @@ class ToyDetectorBenchmark : public benchmark::Fixture {
         apply_propagation_config(finding_cfg.propagation);
         apply_propagation_config(fitting_cfg.propagation);
 
+        // Prevent segmentation fault in the Kalman Fitter
+        fitting_cfg.covariance_inflation_factor = 1.f;
+
+        finding_cfg.chi2_max = 30.f;
```

Comment:
Same here

Reply:
I had to set it that big to recover all tracks in a single muon, no CKF branching scenario.